x/gov/types/v1beta2/msgs.go

```diff
@@ -45,22 +46,14 @@ func (m *MsgSubmitProposal) GetMsgs() ([]sdk.Msg, error) {
 	return sdktx.GetMsgs(m.Messages, "sdk.MsgProposal")
 }
 
-func (m *MsgSubmitProposal) SetInitialDeposit(coins sdk.Coins) {
-	m.InitialDeposit = coins
-}
-
-func (m *MsgSubmitProposal) SetProposer(address fmt.Stringer) {
-	m.Proposer = address.String()
-}
-
-func (m *MsgSubmitProposal) SetMessages(messages []sdk.Msg) error {
-	msgs := make([]*types.Any, len(messages))
+func (m *MsgSubmitProposal) setMessages(messages []sdk.Msg) error {
+	msgs := make([]*codectypes.Any, len(messages))
```

Comment:
you could probably use `SetMsgs` from `types/tx`

Reply:
Done, thanks